### ISSUE TITLE
Nvda doesn't work with flash player installer.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2917]
**Reported by meerkat on 2013-01-08 03:52**
Nvda doesn't work with the flash player installer in windows xp. The installer is found at 

http://helpx.adobe.com/flash-player/kb/installation-problems-flash-player-windows.html

The installer when launched just says flash player installer and nothing else, while on windows 7 it pops up with the lisence aggrement. You can find the flash player installer under the problem marked Progress bar hangs during download.

**Comment by bhavyashah** [comment]
According to https://github.com/nvaccess/nvda/issues/2917#issuecomment-155303798, @brian1gaff seems to think that this issue may be specific to Windows XP. Also, with the demise of Flash, it investigating Flash specific bug reports may no longer be worthwhile. Thus, I suggest closing.

**Comment by ehollig** [comment]
This sounds like a duplicate of #2179. Regardless, this will NeedExternalFix from Adobe. Closing as duplicate

**Comment by Brian1Gaff** [comment]
Well as mentioned, Flash thank goodness is on the way out now. In fact than 
very latest installers do work with all windows, but some of the 
experimental builds they put out had an installer that only seems to work in 
windows 7 and up for nvda, ie the buttons are not identified or missing 
altogether.
 I hope we can safely  put this, and flash into the bin!
Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
